exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,15 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+    return mark >= 80 ?
+        "A" :
+        mark > 60 && mark < 80 ?
+        "B" :
+        mark >= 50 && mark <= 60 ?
+        "C" :
+        "F";
+}
```

Comment:
Good work - you could remove the second checks for B and C. For example, when checking for a B grade you already know that the grade is less than 80, so you could just check that grade > 60